tests/helpers.py

```diff
@@ -2056,3 +2057,200 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+
+def check_required_osd_count(total_osd_nos=3):
+    """
+    Function to check required OSD count and if count is less add OSDs
+
+    Args:
+        total_osd_nos (int): OSD count per node.
+
+    Returns:
+        cluster_total_osd_count (int): Total OSD count of overall cluster.
+
+    """
+    # Make sure setup has required number of OSDs
+    actual_osd_count = cluster.count_cluster_osd()
+    ocs_nodes = node.get_typed_nodes(node_type='worker')
+    expected_osd_count = len(ocs_nodes) * total_osd_nos
+    if actual_osd_count == expected_osd_count:
+        logging.info(f"Setup has OSD count as per OCS workers")
+    else:
+        storage_cluster.add_capacity(int((expected_osd_count - actual_osd_count) / 3))
+        logging.info(f"Now setup has expected osd count {expected_osd_count}")
+    return cluster.count_cluster_osd()
```

Comment:
The intention of this function seems to add osds, so why do you want to return the count. You should expect this count to be there or else the function should assert. wdyt?

Reply:
Since it's library function added return. If test case expects a particular number of OSD to be in cluster then it can be checked with the return value after adding new osd's.